sam/config.py

```diff
@@ -19,3 +25,28 @@
     "GITHUB_REPOS",
     {repo: repo for repo in os.getenv("GITHUB_REPOS", "").split(",") if repo},
 )
+
+
+@dataclass
+class AssistantConfig:
+    name: str
+    assistant_id: str
+    instructions: list[str]
+    project: str
+
+    @cached_property
+    def system_prompt(self):
+        return "\n\n".join(
+            Path(instruction).read_text() for instruction in self.instructions
+        )
+
+
+project = Path("pyproject.toml").resolve()
```

Comment:
Shouldn't this be configurable? I am not sure, that customizing `pyproject.toml` would be a way to go for people using `Sam`.